src/styles/mixins/buttons.scss

```diff
@@ -48,6 +47,8 @@
   padding: 0.5em;
   border-radius: var.$border-radius;
   border: none;
+  transition: color 0.2s ease-out, background-color 0.2s ease-out,
```

Comment:
This gave me pause when I first saw it because I remember that part of the reason that the `transition` was previously applied within the `:not([disabled])` selector rules was to prevent disabled buttons from animating (transitioning) their background colors—as that made disabled buttons look active.

But the disabled buttons I can think of—the publish control submit button when there's no content in an annotation, the markdown-editor buttons when in "preview" mode—seem to behave correctly after this change.

I'm left scratching my head as to _why_ this works (when before it didn't unless it was inside of the not-disabled rule), but it does _seem_ to work.

Reply:
I think it may be from this rule

```
 &:hover:not([disabled]),
  &:focus:not([disabled]) {
    color: var.$color-text--inverted;
    background-color: var.$grey-6;
  }
```

so, the transition still applies when the button is disabled, it's just that the color is fixed